### ISSUE TITLE
Allow having URL as instance contact address

### ISSUE THREAD

**Issue by tcitworld** [tootsuite/mastodon#7483]
We don't want to put an email as a contact address, but an contact platform URL, however since there's the `mailto:` link since https://github.com/tootsuite/mastodon/pull/4450 that breaks everything.
So I would like it to be either
* totally removed
* added only if the string is an email

**Comment by mjankowski** [comment]
Would just putting a URL into the free form about text solve this?

**Comment by tcitworld** [comment]
Nope, as the aim is also to get the contact information through the API (the `contact.email` property of the `/api/v2/instance` endpoint).

Also it still enforces to have a publicly available e-mail which will get spammed.

**Comment by mjankowski** [comment]
Thanks for confirming.

It seems a little odd to accept a url for a field called email -- would adding an (optional) url field help?

**Comment by tcitworld** [comment]
Ideally I would imagine a new agnostic field which could handle any URI type (`mailto:`, `https:`, etc), prefilled with `mailto:` links to the existing `email` properties.

**Comment by mjankowski** [comment]
Thanks - I think "add a `contact.url` field to the api" is way more viable than "accept urls for an email field".

I'll build/add this if someone from core team labels as wanted.
